plugins/workspace-minimap/src/focus_region.ts

```diff
@@ -45,6 +45,10 @@ export class FocusRegion {
     private minimapWorkspace: Blockly.WorkspaceSvg,
   ) {
     this.id = String(Math.random()).substring(2);
+    this.onChangeWrapper = (e: Blockly.Events.Abstract) => {}; 
```

Comment:
Instead of assigning an empty method, can you assign `this.onChange.bind()` here, and remove the later assignment?